lib/sidekiq_unique_jobs/locksmith.rb

```diff
@@ -378,6 +394,14 @@ def lock_info
       )
     end
 
+    def lock_score
+      @lock_score ||= item[CREATED_AT]
```

Comment:
I don't think this is right. It should be the time at which the job is scheduled, and I don't think this is that.